src/matches.js

```diff
@@ -1,8 +1,18 @@
+function assertNotNullOrUndefined(matcher) {
+  if (matcher == null) {
+    throw new Error(
+      `It looks like you passed ${matcher} instead of a matcher. Did you do something like getByText("${matcher}")?`,
```

Comment:
Let's do what we can to avoid language that could be aggravating for sometime struggling to make something work. Thanks!

```suggestion
      `It looks ${matcher} was passed instead of a matcher. Did you do something like getByText("${matcher}")?`,
```

Reply:
Ooh, great tone suggestion, thank you! I love little improvements like that 😄

Reply:
I prefer the grammar of the original, how about this?
```suggestion
      `It looks like ${matcher} was passed instead of a matcher. Did you do something like getByText("${matcher}")?`,
```